docs/api/rest_api/rest_api_reference/input/ez-user-groups.raml

```diff
@@ -127,6 +127,36 @@ get:
             404:
                 description: Error - the User Group does not exist.
     /users:
+        get:
+            displayName: Load Users of Group
+            description: Loads the Users of the Group with the given ID.
```

Comment:
The controller is extracting the ID from the path argument by splitting this path on slashes and keeping the last item ([` extractLocationIdFromPath`](https://github.com/ibexa/rest/blob/v4.3.0/src/lib/Server/Controller/User.php#L1043)). So, it works with just the ID.

The description is not wrong but doesn't cover the whole possibilities. If having only a path, a developer shouldn't extract the ID from this path before use but directly use this path. Anyway, just having access to the path seems like a rare case to me, and an ID simpler to use. 

I propose to not edit this description in this PR.

But this is a more general problem than just below `/user/groups/{path}/`;
Other routes having a {path} argument are in fact extracting the ID from it with their documentation not saying it.
From my PoV, the worst could be [`/content/locations/{path}` description](https://github.com/ezsystems/developer-documentation/blob/6df96089d7247df80266d0edca00d41073791676/docs/api/rest_api/rest_api_reference/input/ez-content-location.raml#L27) only documenting the path while [its controller will extract the ID from this path](https://github.com/ibexa/rest/blob/v4.3.0/src/lib/Server/Controller/Location.php#L392). The 3 following example are equivalent but only the complex one is proposed:
* `/content/locations/1/2/61` (full path)
* `/content/locations/2/61` (incomplete path)
* `/content/locations/61` (ID)